### PR TITLE
20201030 designupdate

### PR DESCRIPTION
**Pull request by dreirik**
* new design for event pages. now it shows on top either
 * a banner if it is an online event with a banner
 * a map if it is an offline event
 * just a divider in case it is online without a banner

* several slight design changes including the contribute box and the presentation of online banners to include links

* some general design, seo and sharing optimization

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 $(shell mkdir -p build/2019 build/2020 build/img build/js build/styles/images)
-SOURCE_IMGS=$(shell find src/img/ -type f -name '*.png' -o -name '*.jpg' -o -name '*.svg')
+# here is the import of all imagees from src/img into the build process, see https://github.com/foss-events/website/pull/179/commits/b695c04ec9eecf9dbda4efe0646cc592a8c746ef
```
Comment (weeman1337) [inline Makefile:2]:
```suggestion
# here is the import of all images from src/img into the build process, see https://github.com/foss-events/website/pull/179/commits/b695c04ec9eecf9dbda4efe0646cc592a8c746ef
```

File: generator/parser.py [hunk 2]
```diff
@@ -161,6 +161,7 @@ def parse_event(row, today):
     event = {
         'id': row['id'],
         'label': row['label'],
+	'name': row['name'],
```
Comment (weeman1337) [inline generator/parser.py:164]:
```suggestion
        'name': row['name'],
```

Comment (weeman1337) [reply generator/parser.py:164]: Ist `name` immer da oder sollte hier besser nen Fallback hin?
`row.get('name', fallback)`

File: src/templates/event.html [hunk 3]
```diff
@@ -34,90 +34,114 @@
...
 <body>
 
-{% include 'partials/header-logo.html' %}
+<!-- start with header -->
 
-<h2 class="event-title">{{ event['label'] }}</h2>
-{% if event['editions_topic'] %}
-    <div class="event-subtitle">This edition's topic: <em>{{ event['editions_topic'] }}</em></div>
-{% endif %}
+	{% include 'partials/header-logo.html' %}
```
Comment (weeman1337) [inline src/templates/event.html:44]:
```suggestion
    {% include 'partials/header-logo.html' %}
```

File: src/templates/event.html [hunk 4]
```diff
@@ -242,32 +266,38 @@ <h2 class="event-title">{{ event['label'] }}</h2>
 	<div class="event-props__group">
         <div class="event-props__label">Registration</div>
         <div class="event-props__content">
-         <a href="{{ event['registration'] }}">{{ event['registration'] }}</a>
+			{{ event['registration'] | url_mixed }}
+			<!-- full url would be <a href="{{ event['registration'] }}">{{ event['registration'] }}</a> -->
```
Comment (weeman1337) [inline src/templates/event.html:270]:
```suggestion
```

File: src/templates/event.html [hunk 5]
```diff
@@ -242,32 +266,38 @@ <h2 class="event-title">{{ event['label'] }}</h2>
...
-        <h3>please <a href="/about.html#contributing">help fix it</a></h3>
-    </div>
-</div>
 
-		{% if event['online'] == false %}
+<!-- the contribute-element -->
+
+    <div class="h1-divider"></div>
+    <div class="text-center text-italic">
+        <p>info missing or info out of date?</p>
```
Comment (weeman1337) [inline src/templates/event.html:282]:
```suggestion
        <p>info missing or out of date?</p>
```

**Review by weeman1337 — APPROVED** [review]

File: generator/parser.py (hunk 2 above)
Comment (dreirik) [reply generator/parser.py:164]: label  und name sind immer da. 'shortname' ist optional

**Pull request merged**